nodejs/week3/index.js

```diff
@@ -0,0 +1,95 @@
+import "dotenv/config";
+import express from "express";
+import cors from "cors";
+import bodyParser from "body-parser";
+import knex from "./database_client.js";
+import nestedRouter from "./routers/nested.js";
+import mealsRouter from "./routers/meals.js";
+import reviewsRouter from "./routers/reviews.js";
+import reservationsRouter from "./routers/reservations.js";
+
+
+
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.json());
+
+const apiRouter = express.Router();
+
+// You can delete this route once you add your own routes
+apiRouter.get("/", async (req, res) => {
+  const SHOW_TABLES_QUERY =
+    process.env.DB_CLIENT === "pg"
+      ? "SELECT * FROM pg_catalog.pg_tables;"
+      : "SHOW TABLES;";
+  const tables = await knex.raw(SHOW_TABLES_QUERY);
+  res.json({ tables });
+});
+
+// This nested router example can also be replaced with your own sub-router
+apiRouter.use("/reservations", reservationsRouter);
+apiRouter.use("/reviews", reviewsRouter);
+apiRouter.use("/meals", mealsRouter);
+apiRouter.use("/nested", nestedRouter);
+app.use("/api", apiRouter);
+
+
+app.listen(process.env.PORT, () => {
+  console.log(`API listening on port ${process.env.PORT}`);
+});
+
+app.get("/my-route", (req, res) => {
+  res.send("Hi friend");
+});
+
+//*/future-meals	Respond with all meals in the future (relative to the when datetime)//*
+app.get("/future-meals", async (req, res) => {
+
+  const GET_MEALS_QUERY = "SELECT * FROM meals WHERE when > NOW();";
+  const meals = await knex.raw(GET_MEALS_QUERY);
+  res.json({ meals });
```

Comment:
The variable meals in this case is an array of two objects: the first element is the actual result, (array of databae rows as JS objects) and the second one is a string that represents the schema of the table. You should only return the first element: the array. 
```suggestion
  const [meals, schema] = await knex.raw(GET_MEALS_QUERY);
  res.json({ meals });
```